packages/blocks/src/api/factory.js

```diff
@@ -378,7 +448,7 @@ export function switchToBlockType( blocks, name ) {
 		return null;
 	}
 
-	return transformationResults.map( ( result, index ) => {
+	const rtn = transformationResults.map( ( result, index ) => {
```

Comment:
Minor: I'm not sure why this was changed, but in any case, I'd tend to discourage abbreviations since they almost never provide any meaningful benefit (and conversely almost always add some ambiguity or confusion) over their more verbose counterparts.

Reply:
My bad. Introduced during debugging. Reverted